### PR TITLE
[ENG-4944] - ad refresh

### PR DESCRIPTION
**Pull request by agrecoFS**

### CODE REVIEW

**Review by aecook — APPROVED** [review]

**Comment by premesh-freestar** [comment]
Need to bump the react version @aforwark  can u create a new release version in jira to correspond to this as well ?

**Comment by agrecoFS** [comment]
@premesh-freestar - the version bump happens by running `npm version` command which i have historically done after things are merged, then we publish from master. we can change it to happen sooner if you'd like

**Comment by premesh-freestar** [comment]
@agrecoFS can u update this to resolve conflicts ?  i think with the new code changes u can implement a refreshPlacement method in the freestarWrapper.js that you can then call like ive done for display and delete ? cc @aecook

File: src/components/FreestarAdSlot/freestarWrapper.js [hunk 1]
```diff
@@ -1,74 +1,81 @@
...
         onDeleteAdSlotsHook(placementName)
+      }
     })
   }
 
-  setPageTargeting( key, value) {
-    window.freestar = window.freestar || {};
+  refreshAdSlot (placementName, onAdRefreshHook) {
+    window.freestar.queue.push(() => {
+      window.freestar.newAdSlots()
```
Comment (premesh-freestar) [inline src/components/FreestarAdSlot/freestarWrapper.js:51]: i believe there is a freestar.refreshAdSlot api to use here and need to pass in the placement name i believe ?

**Review by premesh-freestar — APPROVED** [review]

**Pull request merged**